### DISCUSSION TITLE
Is the default `I100` equivalent to `isort --profile=black`?

### DISCUSSION THREAD

**Discussion by failable** [astral-sh/ruff#12577] [Q&A] [answered]
Is the default `I100` equivalent to `isort --profile=black`? If not, what is the equivalent of `isort --profile=black`?

Thanks.

**Comment by failable** [comment, marked as answer]
Found this: https://docs.astral.sh/ruff/faq/#how-does-ruffs-import-sorting-compare-to-isort .

**Reply by charliermarsh** [reply]
Yes, it's designed to be equivalent to `--profile=black`.
